backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java

```
@@ -139,7 +139,8 @@ public SetupEntity getSetupByTitle(UserEntity authUser, String title) {
                 return setupEntity;
             }
         }
-        throw new EntityNotFoundException(String.format("no setup with title=%s !",title ));
+        //throw new EntityNotFoundException(String.format("no setup with title=%s !",title ));
+        return SetupEntity.builder().build();
```

This doesn't look right.. I think you should throw an exception here